### PR TITLE
node-tests

### LINKED ISSUE #356: Nodes section - Functional Tests
Tests that will cover all of the node's tab functionalities.

### PR DESCRIPTION
**Pull request by emedrano2112**
Closes #356

### CODE REVIEW

File: func-tests/Nodes/general-nodes-information.feature [hunk 1]
```diff
@@ -0,0 +1,584 @@
...
+    Given that the user is in the Pocket Dashboard. Example: 
+    |https://dashboard.testnet.pokt.network/dashboard|
+    When the page is fully loaded.
+    And user clicks on "Nodes" from the left pannel.
+    And user is redirected to the nodes section:
+    |https://dashboard.testnet.pokt.network/dashboard/nodes|
+    And the user clicks on "create" top-right.
+    Then the user should be taken to the node's creation form:
+    |https://dashboard.testnet.pokt.network/dashboard/nodes/new|
+    And provide a "Node Name", "Node operator or Company Name" and "Contact Email", notice that these fiels are required.
```
Comment (nymd) [inline func-tests/Nodes/general-nodes-information.feature:226]: fiels == fields

Comment (nymd) [reply func-tests/Nodes/general-nodes-information.feature:226]: typo is copy/pasted a few times

File: func-tests/Nodes/general-nodes-information.feature [hunk 2]
```diff
@@ -0,0 +1,584 @@
...
+    And click on "continue".
+    Then user shuld be takent to the passphrase creation step:
```
Comment (nymd) [inline func-tests/Nodes/general-nodes-information.feature:228]: takent == taken

Comment (nymd) [reply func-tests/Nodes/general-nodes-information.feature:228]: also copy/pasted

File: func-tests/Nodes/general-nodes-information.feature [hunk 3]
```diff
@@ -0,0 +1,584 @@
...
+    |https://dashboard.testnet.pokt.network/dashboard/nodes/new/passphrase|
+    And see a successfull POST (200 - true) to:
```
Comment (nymd) [inline func-tests/Nodes/general-nodes-information.feature:230]: successfull == successful

Comment (nymd) [reply func-tests/Nodes/general-nodes-information.feature:230]: also copy/pasted

**Review by nymd — CHANGES_REQUESTED** [review]

File: func-tests/Nodes/general-nodes-information.feature (hunk 1 above)
Comment (emedrano2112) [reply func-tests/Nodes/general-nodes-information.feature:226]: Fixed.

File: func-tests/Nodes/general-nodes-information.feature (hunk 3 above)
Comment (emedrano2112) [reply func-tests/Nodes/general-nodes-information.feature:230]: Fixed.

Comment (emedrano2112) [reply func-tests/Nodes/general-nodes-information.feature:230]: Fixed.

**Pull request merged**